policy/modules/system/init.te

```diff
@@ -56,6 +56,12 @@ domain_entry_file(init_t, init_exec_t)
 kernel_domtrans_to(init_t, init_exec_t)
 role system_r types init_t;
 
+#
+# init_tmp_t is the type for /tmp/systemd-private-xxxx
+#
```

Comment:
bad idea, unless you transition back to tmp_t on dirs with name "tmp"

ie:

`filetrans_pattern(init_t, init_tmp_t, tmp_t, dir, "tmp")`

You see these directories will be bind mounted to the service /tmp and that would then mean that any service that has PrivateTmp=yes and that maintains objects in "/tmp" will have to be able to write to i`nit_tmp_t` instead of `tmp_t`